src/stores/RootStore.ts

```diff
@@ -37,7 +38,8 @@ export default class RootStore {
 
```

Comment:
After successful navigation via url, url stays the same and never updates. This might be confusing. We should delete it as soon as we parsed it.